pom.xml

```diff
@@ -76,6 +76,12 @@
             <version>0.0.5</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Why add this dependency? I don't want to bring in any dependencies unless absolutely necessary.